crates/host/src/wasmbus/handler.rs

```diff
@@ -545,7 +593,13 @@ impl Bus for Handler {
             tx.await.context("failed to transmit parameters")?;
             Ok(results)
         } else {
-            bail!("component attempted to invoke a function on an unknown target")
+            bail!(
+                "component [{}] attempted to invoke a function [{}/{}] on an unknown target [{}]",
+                self.component_id,
+                instance,
+                name,
+                target.lattice_id().unwrap_or("<unknown>"),
```

Comment:
Shouldn't this target be the target "id", rather than the lattice ID?

Reply:
Or, after reading the implementation of the method, this is not the _lattice name_ this is the invocation target on the lattice, right? If I understand correctly this LGTM

Reply:
Ah this is not the lattice name -- it's the "reference by which the entity can be addressed on the lattice" becuase that is off the `TargetEntity`. 

This is confusing though, so I'll change this!